src/main/java/core/basesyntax/resources/inputDataForPeriod.csv

```diff
@@ -0,0 +1,8 @@
+    b,banana,20
+    b,apple,100
+    s,banana,100
+    p,banana,13
+    r,apple,10
+    p,apple,20
+    p,banana,5
+    s,banana,50
```

Comment:
Thas does not look like a proper path
![image](https://user-images.githubusercontent.com/80981426/212966063-66160f13-706d-4c63-b0e5-75ed91ccafaa.png)